src/components/PopupwithAvatar.js

```diff
@@ -0,0 +1,47 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithAvatar extends Popup {
```

Comment:
Please, delete `PopupWithAvatar`. You need to use `PopupWithForm` for any modal with a form. You don't need to duplicate it

Reply:
PopupwithAvatar has been fully removed